examples/terraform/openstack/variables.tf

```
@@ -88,6 +88,12 @@ variable "control_plane_flavor" {
   type        = string
 }
 
+variable "control_plane_group_policy" {
+  description = "OpenStack instance group policy for the control plane nodes. Cannot be changed without recreating VMs!"
```

```suggestion
  description = "OpenStack instance group policy for the control plane nodes"
```

We'll document the other part in the changelog.